src/main/java/com/refinedmods/refinedstorage/apiimpl/network/grid/handler/ItemGridHandler.java

```diff
@@ -82,11 +90,22 @@ public void onExtract(ServerPlayerEntity player, UUID id, int flags) {
         if (!took.isEmpty()) {
             if ((flags & EXTRACT_SHIFT) == EXTRACT_SHIFT) {
                 IItemHandler playerInventory = player.getCapability(CapabilityItemHandler.ITEM_HANDLER_CAPABILITY, Direction.UP).orElse(null);
```

Comment:
Can't we do a `ifPresent`, much cleaner.

Reply:
If I put it into a lambda I need to make size final and convert took into an array :/

changed to resolve which is a bit cleaner